aleph-client/src/multisig.rs

```diff
@@ -0,0 +1,481 @@
+use std::collections::HashSet;
+
+use anyhow::{ensure, Result};
+use codec::{Decode, Encode};
+use log::error;
+use primitives::Balance;
+use sp_core::{blake2_256, crypto::AccountId32, Pair};
+use substrate_api_client::{compose_extrinsic, XtStatus::Finalized};
+use thiserror::Error;
+
+use crate::{
+    account_from_keypair, try_send_xt, AccountId, BlockNumber, Connection, KeyPair,
+    UncheckedExtrinsicV4, H256,
+};
+
+/// `MAX_WEIGHT` is the extrinsic parameter specifying upperbound for executing approved call.
+/// Unless the approval is final, it has no effect. However, if due to your approval the
+/// threshold is reached, you will be charged for execution process. By setting `max_weight`
+/// low enough, you can avoid paying and left it for another member.
+///
+/// However, passing such parameter everytime is cumbersome and introduces the need of either
+/// estimating call weight or setting very high universal bound at every caller side.
+/// Thus, we keep a fairly high limit, which should cover almost any call (0.05 token).
+const MAX_WEIGHT: u64 = 500_000_000;
+
+/// Gathers all possible errors from this module.
```

Comment:
nit: lie :P https://github.com/paritytech/substrate/blob/de8b34c814089e9561cbc1cdb4399bfc91d4cd46/frame/multisig/src/lib.rs#L175

Reply:
Ok, `from this module` was misleading for me. I though you mentioned the multisig module